blues/formats.py

```diff
@@ -229,17 +342,16 @@ def write(self, coordinates, parameters=None, environment=None,
 
         except self.tables.NoSuchNodeError:
             raise ValueError("The file that you're trying to save to doesn't "
-                "contain the field %s. You can always save a new trajectory "
-                "and have it contain this information, but I don't allow 'ragged' "
-                "arrays. If one frame is going to have %s information, then I expect "
-                "all of them to. So I can't save it for just these frames. Sorry "
-                "about that :)" % (name, name))
+                             "contain the field %s. You can always save a new trajectory "
+                             "and have it contain this information, but I don't allow 'ragged' "
+                             "arrays. If one frame is going to have %s information, then I expect "
+                             "all of them to. So I can't save it for just these frames. Sorry "
```

Comment:
An error message which apologizes?

Reply:
This was copied from the HDF5Trajectory class from MDTraj :)